argo_stage/schema.xml

```diff
@@ -68,99 +44,89 @@
+    <dynamicField name="*_tsiv"  type="text" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_tsimv" type="text" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
 
-    <!-- location (_ll...) -->
-    <dynamicField name="*_lli"   type="location" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_llim"  type="location" stored="true" indexed="true"  multiValued="true"/>
-    <dynamicField name="*_lls"   type="location" stored="true" indexed="false" multiValued="false"/>
-    <dynamicField name="*_llsm"  type="location" stored="true" indexed="false" multiValued="true"/>
-    <dynamicField name="*_llsi"  type="location" stored="true" indexed="true"  multiValued="false"/>
-    <dynamicField name="*_llsim" type="location" stored="true" indexed="true"  multiValued="true"/>
+    <!-- English text (_te...) -->
+    <dynamicField name="*_tei"    type="text_en" stored="false" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_teim"   type="text_en" stored="false" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_tes"    type="text_en" stored="true" indexed="false" multiValued="false"/>
+    <dynamicField name="*_tesm"   type="text_en" stored="true" indexed="false" multiValued="true"/>
+    <dynamicField name="*_tesi"   type="text_en" stored="true" indexed="true"  multiValued="false"/>
+    <dynamicField name="*_tesim"  type="text_en" stored="true" indexed="true"  multiValued="true"/>
+    <dynamicField name="*_teiv"   type="text_en" stored="false" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_teimv"  type="text_en" stored="false" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_tesiv"  type="text_en" stored="true" indexed="true"  multiValued="false" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_tesimv" type="text_en" stored="true" indexed="true"  multiValued="true"  termVectors="true" termPositions="true" termOffsets="true"/>
 
-    <!-- you must define copyField source and dest fields explicity or schemaBrowser doesn't work -->
-    <field name="all_text_timv" type="text" stored="true" indexed="true" multiValued="true" termVectors="true" termPositions="true" termOffsets="true"/>
+    <dynamicField name="*_text_nostem_i"  type="textNoStem" indexed="true"  stored="false" multiValued="false"/>
+    <dynamicField name="*_text_nostem_im"  type="textNoStem" indexed="true"  stored="false" multiValued="true"/>
 
-    <!-- for structured values (JSON blobs) -->
-    <dynamicField name="*_struct" type="string" stored="true" indexed="false" multiValued="true" omitNorms="true" />
-    
```

Comment:
this may be a nice idea but we're not using this.  If we need it, we'll bring it back.